packages/run-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -101,12 +101,24 @@ const makeVaultInvitation = ({ state }) => {
     });
     const {
       give: { Collateral: collateralAmount },
+      want: { RUN: borrowAmount },
     } = seat.getProposal();
     const { brand: brandIn } = collateralAmount;
     assert(
       collateralTypes.has(brandIn),
       X`Not a supported collateral type ${brandIn}`,
     );
+
+    assert(
+      AmountMath.isGTE(
+        borrowAmount,
+        // @ts-expect-error VaultDirectoryParams isn't right
+        state.directorParamManager.getMinInitialLoan(),
+      ),
+      // @ts-expect-error VaultDirectoryParams isn't right
+      X`Minimum loan amount is ${state.directorParamManager.getMinInitialLoan()}, won't lend ${borrowAmount}`,
```

Comment:
`The requested amount (${requestedAmount}) must be at least ${state.directorParamManager.getMinInitialLoan()}`

Reply:
I went with `The request must be for at least ${getMinInitialDebt}. ${requestedAmount} is too small`